.circleci/config.yml

```diff
@@ -23,10 +28,58 @@ jobs:
           $job = start-job {dotnet run --project C:\Users\circleci\project\MockServer\MockServer}
           Start-Sleep 10
           .\Tests\doc-downloader.tests.ps1
+          .\Tests\drive-axle-doc-downloader.tests.ps1
       - run: |
           Write-Host (Split-Path -Path $PSScriptRoot)
           $ProgressPreference = 'SilentlyContinue';
           Compress-Archive -Path "C:\Users\circleci\project\README.txt", "C:\Users\circleci\project\configurations.json", "C:\Users\circleci\project\*.ps1" -DestinationPath document-downloader.zip
       - store_artifacts:
           path: document-downloader.zip
           destination: document-downloader.zip
+
+jobs:
+  build_powershell_five:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run_tests
+  build_powershell_six:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - powershell_version_setup:
+          powershell_setup_steps:
+            - run: |
+                choco install powershell-core
+                choco uninstall powershell-core
+                choco install powershell-core --version=6.2.4
+                cd 'C:\Program Files\PowerShell\6'
+                .\pwsh.exe
+      - run_tests
+  build_powershell_seven:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - powershell_version_setup:
+          powershell_setup_steps:
+            - run: |
+                choco install powershell-core
```

Comment:
This will probably implicitly become 8 at some point, what do you think about adding an explicit --version to this too?